test/config/TestConfig.cpp

```diff
@@ -21,6 +21,7 @@ using namespace Aws::Iot::DeviceClient;
 using namespace Aws::Iot::DeviceClient::Util;
 
 const string filePath = "/tmp/aws-iot-device-client-test-file";
+const string invalidFilePath = "/tmp/invalid-file-path";
```

Comment:
Please add a `std::remove(invalidFilePath)` on the `ConfigTestFixture::SetUp` method to be sure.